Kuadrant/multicluster-gateway-controller
# Gh 228

## Verification 

1. run make local-setup OCM_SINGLE=true MGC_WORKLOAD_CLUSTERS_COUNT=1    
1. ensure a secret called mgc-aws-credentials in the namespace  multi-cluster-gateways is created with the creds from aws-credentials.env
2. ensure the managed zone cr created contains a reference in the spec to the name and namespace and type of the dnsprovider credentials from above 
```
spec:
   provider:
    name: mgc-aws-credentials
    namespace: multi-cluster-gateways
    type: AWS

```
1. run through the walkthrough https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/docs/how-to/ocm-control-plane-walkthrough.md everything should work as expected
2. Status in the managedzone should be updated
